### DISCUSSION TITLE
A dvd, 484 days to encode and counting.........

### DISCUSSION THREAD

**Discussion by SpeedLee69** [community/community#19265] [Other Feature Feedback, Questions, & Ideas]
Having downloaded/Installed an updated version of Vidcoder, why does it indicate that it will take 484 days to encode a dvd that is only 97 minutes long and is still stuck on 0.2% encoded.  This happens with every dvd I'm trying to convert.
Sorry guys, I don't understand code speak and plain English suits me better.

Incidentally, an older version ( 3. something-I think)  on my old but slow laptop seems to be unaffected.

aaaaaaaaargh!!

 

![Freeway](https://user-images.githubusercontent.com/92914159/175270226-4620e2e2-9cef-4229-b48f-6656d43fd722.PNG)

**Comment by JohnVeness** [comment]
Hello! You're asking in the wrong place. Assuming this is the same Vidcoder, I think you should ask in https://github.com/RandomEngy/VidCoder/discussions
